oci/client/build_test.go

```diff
@@ -61,6 +61,12 @@ func TestBuild(t *testing.T) {
 			ignorePath: []string{"/*", "!/internal"},
 			checkPaths: []string{"/testdata", "!internal/", "build.go", "meta.go"},
 		},
+		{
+			name:       "existing file",
+			path:       "testdata/artifact",
```

Comment:
This doesn’t look like it tests the new feature. We need to pass a file and check that only that one is in the tarball

Reply:
According to that, I've updated the `checkPathExists` function to be able to work with files in addition to directories